### DISCUSSION TITLE
install instructions currently assume ROS installed

### DISCUSSION THREAD

**Discussion by htp2** [WPI-AIM/ambf#200] [General]
Currently, the wiki instructions for building AMBF assume you have ROS installed (they tell you to source a devel folder that doesn't exist if you don't have ROS installed). 

This is perfectly fine, but maybe we could add something to the Wiki that says this explicitly. And also clarifies that ROS is needed to use the AMBF python client.

**Comment by adnanmunawar** [comment]
Hi Henry, thanks for this discussion, yes this is an oversight. I believe that #202 may be related. I shall updated the Readme.

**Comment by adnanmunawar** [comment]
I updated the Readme to clarify the ROS installation part.

**Reply by htp2** [reply]
Awesome! Thanks Adnan
